api/v1alpha1/maintenanceoperatorconfig_types.go

```diff
@@ -0,0 +1,79 @@
+/*
+ Copyright 2024, NVIDIA CORPORATION & AFFILIATES
+
+ Licensed under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+
+     http://www.apache.org/licenses/LICENSE-2.0
+
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+*/
+
+package v1alpha1
+
+import (
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/util/intstr"
+)
+
+// OperatorLogLevel is the operator log level. one of: ["debug", "info", "error"]
+// +kubebuilder:validation:Enum=debug;info;error
+type OperatorLogLevel string
+
+// MaintenanceOperatorConfigSpec defines the desired state of MaintenanceOperatorConfig
+type MaintenanceOperatorConfigSpec struct {
+	// MaxParallelOperations indicates how many nodes can be processed in parallel, 0 means no limit
+	// value can be an absolute number (ex: 5) or a percentage of total nodes in the cluster (ex: 10%).
+	// absolute number is calculated from percentage by rounding up.
+	// default: 1
+	// +kubebuilder:default=1
+	// +kubebuilder:validation:XIntOrString
+	MaxParallelOperations *intstr.IntOrString `json:"maxParallelOperations,omitempty"`
```

Comment:
From the doc strings it is not clear how `MaxParallelOperations` and `MaxUnavailable` work together. Can you clarify this in the doc strings for these fields.

Reply:
will do.